### ISSUE TITLE
[a11y] CheckboxText - No focus visible

### ISSUE THREAD

**Issue by fffan64** [AxaFrance/design-system#1227]
![Image](https://github.com/user-attachments/assets/fc3e4d9a-0ccf-411d-b583-9af5955d6da5)

**Comment by pplancq** [comment]
@fffan64 normalement les CheckboxText du package apollo en theme apollo et LF on un focus visible pour la navigation clavier, ce qui doit corriger cette issue.
peux tu confirmer que c'est ok pour toi.
